pkg/chaosctl/common/common.go

```diff
@@ -407,18 +414,62 @@ Please check network policy / firewall, or see FAQ on website`, daemon.Name, dae
 	return nil
 }
 
-// NewChaosDaemonClientLocally would create ChaosDaemonClient in localhost
-func NewChaosDaemonClientLocally(port int, caCert string, cert string, key string) (daemonClient.ChaosDaemonClientInterface, error) {
+// ConnectToLocalChaosDaemon would connect to ChaosDaemon run in localhost
+func ConnectToLocalChaosDaemon(port int) (daemonClient.ChaosDaemonClientInterface, error) {
 	if cli := mock.On("MockChaosDaemonClient"); cli != nil {
 		return cli.(daemonClient.ChaosDaemonClientInterface), nil
 	}
 	if err := mock.On("NewChaosDaemonClientError"); err != nil {
 		return nil, err.(error)
 	}
+	cc, err := getGrpcClient(port)
 
-	cc, err := grpcUtils.CreateGrpcConnection("localhost", port, caCert, cert, key)
 	if err != nil {
 		return nil, err
 	}
 	return daemonClient.New(cc), nil
 }
+
+func getGrpcClient(port int) (*grpc.ClientConn, error) {
+	if TLSFiles.CaCert == "" || TLSFiles.Cert == "" || TLSFiles.Key == "" {
+		PrettyPrint("TLS Files are not complete, fall back to use secrets.", 0, Green)
+		config, err := getTLSConfigFromSecrets()
+		if err != nil {
+			return nil, err
+		}
+		return grpcUtils.CreateGrpcConnectionFromRaw("localhost", port, config.caCert, config.cert, config.key)
+	}
+	PrettyPrint("Using TLS Files.", 0, Green)
+	return grpcUtils.CreateGrpcConnection("localhost", port, TLSFiles.CaCert, TLSFiles.Cert, TLSFiles.Key)
+}
```

Comment:
I think we only need `grpcUtils.CreateGrpcConnectionFromRaw`, you could make a method called `getTLSConfigFromFiles()`;

Reply:
`grpcUtils.CreateGrpcConnection` is used elsewhere, I cannot simply delete it. But I can make it packed up around `grpcUtils.CreateGrpcConnectionFromRaw`